lib/vite.config.ts

```diff
@@ -13,11 +13,14 @@ export default defineConfig(() => {
           format === "es" ? `${libName}.mjs` : `${libName}.js`,
       },
       rollupOptions: {
-        external: ["react", "axios"],
+        external: ["react"],
         output: {
           globals: { react: "React" },
         },
       },
     },
+    test: {
+      setupFiles: ["./setup.js"],
+    },
```

Comment:
We don't need this. This was for Vitest but we don't use it here

Reply:
I asked @schabibi1 about this line on private 😄

Reply:
Done @alexjoverm ✅ @fgiuliani Responded 😉